.pre-commit-config.yaml

```diff
@@ -13,6 +13,8 @@ repos:
     rev: v4.0.0-alpha.4
     hooks:
       - id: prettier
+        exclude_types:
+          - markdown
```

Comment:
Why are we ignoring markdown now?

Reply:
because we use MyST, which is a bespoke Markdown dialect instead of standard markdown, and therefore not supported by Prettier